### DISCUSSION TITLE
How to manage teams in Dispatch

### DISCUSSION THREAD

**Discussion by rilutham** [Netflix/dispatch#895] [Q&A]
We are seeing some database/column about team-individual relation and also a panel in the incident dashboard named "Primary Teams". However, we cannot find UI to manage the teams. Is there any way to manage it?

**Comment by mvilanova** [comment]
The primary teams in the dashboard are the teams that incident participants belong to. The only team-related management option we offer is at /contact/teams, where you can add teams and define when they should be engaged. If you add a team, its email address will get added to the notifications group on incident creation, so they can receive notifications such as executive reports.
